src/columnstore/columnstore_metadata.cpp

```diff
@@ -20,19 +20,35 @@ extern "C" {
 
 namespace duckdb {
 
-Oid Mooncake() {
-    return get_namespace_oid("mooncake", false /*missing_ok*/);
-}
+namespace {
```

Comment:
The general practice is to use `static` or anonymous namespace for impl file only definitions, so symbols won't be leaked outside of the translation unit.